fission/src/mirabuf/MirabufParser.ts

```diff
@@ -478,7 +460,7 @@ export class Graph {
     }
 
     public AddEdgeDirected(nodeA: string, nodeB: string) {
-        if (!this._adjacencyMap.has(nodeA) || !this._adjacencyMap.has(nodeB)) throw new Error("Nodes aren't in graph")
+        if (!this._adjacencyMap.has(nodeA) || !this._adjacencyMap?.has(nodeB)) throw new Error("Nodes aren't in graph")
```

Comment:
Should both calls to `this._adjacencyMap.has(...)` be nullable (`?`)?

Reply:
No, neither of them should be

Reply:
```suggestion
        if (!this._adjacencyMap.has(nodeA) || !this._adjacencyMap.has(nodeB)) throw new Error("Nodes aren't in graph")
```